src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -535,6 +573,8 @@ g.test('texture_size,3d_texture,uncompressed_format')
     }, !success);
   });
 
+// Compressed formats are not supported in 3D in WebGPU v1 because they are complicated but not very useful.
+/*
 g.test('texture_size,3d_texture,compressed_format')
```

Comment:
According to the discussion at https://github.com/gpuweb/gpuweb/issues/1522, we will not support compressed formats 3D texture. Dawn project follows this rule too. So comment out this test. It spent some time on it, it might be useful if we decided to support compressed formats for 3D texture. So I didn't remove this test. 

If you think you should remove this test. I am fine to do so.

Reply:
If we want to keep the test around (I think it's good, I _think_ we will probably eventually add 3d compressed textures) then I would prefer to keep it uncommented so the code doesn't get too stale. The skip at the beginning is OK I think. Running the skipped test should only waste a few milliseconds at most.

Reply:
You are right. We have already discussed this before. I missed the skip() part below and thought I didn't do that.

Reply:
No worries, I'd have forgotten too but I just happened to see the skip() line.